libraries/reconstruction_cuda/cuda_shift_aligner.cpp

```diff
@@ -80,7 +80,7 @@ template<typename T>
 void CudaShiftAligner<T>::load2DReferenceOneToN(const std::complex<T> *h_ref) {
     auto isReady = (m_isInit && (AlignType::OneToN == m_type));
     if ( ! isReady) {
-        REPORT_ERROR(ERR_LOGICAL_ERROR, "Not ready to load a reference signal");
+        REPORT_ERROR(ERR_LOGIC_ERROR, "Not ready to load a reference signal");
```

Comment:
I'm surprised there's no conflict. This is already in the [devel](https://github.com/I2PC/xmipp/blob/1ca27833e580b8413ae5b86ebddd60fa151a6614/libraries/reconstruction_cuda/cuda_shift_aligner.cpp#L83) branch.